statchatexpress/models/user.js

```diff
@@ -16,4 +16,12 @@ export const User = sequelize.define("User", {
     type: DataTypes.STRING,
     allowNull: false,
   },
+  Gamertag: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
+  Platform: {
+    type: DataTypes.STRING,
```

Comment:
This model design will cause problems when we integrate additional games.

This is defining a one to one model. (One user to one gamertag/platform).
1. What if a user plays the same game on multiple platforms?
2. What if the user plays multiple games with different handles?
3. How are we planning to differentiate games? We know that your new API has totally different statistics available.